include/boost/windows_sspi/context.hpp

```diff
@@ -29,41 +31,34 @@ class sspi_handshake;
 
 class context : public context_base {
 public:
-  using native_handle_type = CredHandle*;
-  using error_code = boost::system::error_code;
-
   explicit context(method)
     : m_impl(std::make_unique<detail::context_impl>())
     , m_verify_mode(verify_none) {
   }
 
-  native_handle_type native_handle() const {
-    return m_impl->handle();
-  }
-
   void add_certificate_authority(const net::const_buffer& ca, boost::system::error_code& ec) {
-    m_impl->add_certificate_authority(ca, ec);
+    try {
```

Comment:
`const_buffer` is usually passed by value

Reply:
It isn't in the existing `boost::asio::ssl` code and I'm trying to use the existing interface as much as possible:

https://www.boost.org/doc/libs/1_74_0/doc/html/boost_asio/reference/ssl__context/add_certificate_authority.html

Reply:
Well, the asio code is wrong :)

Reply:
It most likely is. I actually think the `boost::asio::ssl` interface should be cleaned up quite a bit, but this is one of the very minor things :-)